ArchiMetrics.Analysis/Metrics/TypeMetricsCalculator.cs

```diff
@@ -41,7 +41,8 @@ public TypeMetricsCalculator(SemanticModel semanticModel, Solution solution)
 			var source = CalculateClassCoupling(type, memberMetrics);
 			var depthOfInheritance = CalculateDepthOfInheritance(type);
 			var cyclomaticComplexity = memberMetrics.Sum(x => x.CyclomaticComplexity);
-			var linesOfCode = memberMetrics.Sum(x => x.LinesOfCode);
+			var linesOfCode = memberMetrics.Sum(x => x.LinesOfCode);
+            var sourceLinesOfCode = memberMetrics.Sum(x => x.SourceLinesOfCode);
```

Comment:
If you do want to calculate SLOC, then this is incorrect as you are not capturing the class declaration lines.